.gitignore

```diff
@@ -0,0 +1,138 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
not sure we need all of this, but I guess it will not hurt

Reply:
I used an existing .gitignore template for python